### ISSUE TITLE
Is there a docker image repo for the new version?

### ISSUE THREAD

**Issue by famewolf** [nzbget-ng/nzbget#34]
I'm currently running nzbget on my Truenas.  I didn't see anything mentioned about docker?

**Comment by NineBall8** [comment]
https://hub.docker.com/r/jscottelblein/nzbget This should work I think.

**Comment by pannal** [comment]
Hmm, I'm using this docker image now. NZBget shows `nzbget 21.2-testing-r2333 server-mode` - is that correct?

**Comment by Mouton99** [comment]
I am using [my own docker image](https://hub.docker.com/r/mouton99/nzbget-ng) (built using [this fork or github.com/linuxserver/docker-nzbget](https://github.com/Mouton99/docker-nzbget-ng)), and I see `Currently installed: 21.3-rc1-12-ga70a383c`

[jscottelblein/nzbget](https://hub.docker.com/r/jscottelblein/nzbget) was not updated for 6 months, so it contains a version from 6 months ago.

**Comment by pannal** [comment]
My hero, thank you! `nzbget 21.3-rc1-12-ga70a383c server-mode`

**Comment by famewolf** [comment]
I'll add my thanks as well!

**Comment by fabiorzfreitas** [comment]
> I am using [my own docker image](https://hub.docker.com/r/mouton99/nzbget-ng) (built using [this fork or github.com/linuxserver/docker-nzbget](https://github.com/Mouton99/docker-nzbget-ng)), and I see `Currently installed: 21.3-rc1-12-ga70a383c`

@Mouton99 ,

Hi, thank you for providing this image! But it has a serious issue: it doesn't seem to be able to detect architectures correctly. I tried using the same syntax from linuxserver, mouton99/nzbget-ng:arm64v8-latest (which I tested against their image and it worked properly), but it's not available on your image.

I'm running everything from a Raspeberry Pi 4B 8GB, using both Portainer and Docker CLI to make sure this is correct.

I checked both repos for occurences of "arm64v8" and there were only two matches (besides the readme): Dockerfile.aarch64, which was a perfect match between the 2 repos, and the Jenkinsfile, which had a few different lines.

I would have submitted a PR, but since I still have very little knowledge on Docker, I figured it would be better to come back here, where I found your image!

**Comment by gboudreau** [comment]
> @Mouton99 ,
> 
> Hi, thank you for providing this image! But it has a serious issue: it doesn't seem to be able to detect architectures correctly. I tried using the same syntax from linuxserver, mouton99/nzbget-ng:arm64v8-latest (which I tested against their image and it worked properly), but it's not available on your image.

I am indeed only building and publishing for `linux/arm64`, not any other arch.

What you can do is simply git clone [my fork](https://github.com/Mouton99/docker-nzbget-ng), then run `docker build -t nzbget-ng .` from within that folder. That will give you a local image that you can use with `docker run ... nzbget-ng`

**Comment by Mouton99** [comment]
> Hi, thank you for providing this image! But it has a serious issue: it doesn't seem to be able to detect architectures correctly. I tried using the same syntax from linuxserver, mouton99/nzbget-ng:arm64v8-latest (which I tested against their image and it worked properly), but it's not available on your image.

I am indeed only building and publishing for `linux/arm64`, not any other arch.

What you can do is simply git clone [my fork](https://github.com/Mouton99/docker-nzbget-ng), then run `docker build -t nzbget-ng .` from within that folder. That will give you a local image that you can use with `docker run ... nzbget-ng`

**Comment by famewolf** [comment]
> > Hi, thank you for providing this image! But it has a serious issue: it doesn't seem to be able to detect architectures correctly. I tried using the same syntax from linuxserver, mouton99/nzbget-ng:arm64v8-latest (which I tested against their image and it worked properly), but it's not available on your image.
> 
> I am indeed only building and publishing for `linux/arm64`, not any other arch.
> 
> What you can do is simply git clone [my fork](https://github.com/Mouton99/docker-nzbget-ng), then run `docker build -t nzbget-ng .` from within that folder. That will give you a local image that you can use with `docker run ... nzbget-ng`

I wish you'd mentioned that before...I'm guessing then mine is not pulling even though pointed to your build and I can't do a docker build on a truenas scale install which uses kubernetes for managing containers.  Guess I'll point it back to the original.

**Comment by fabiorzfreitas** [comment]
> > Hi, thank you for providing this image! But it has a serious issue: it doesn't seem to be able to detect architectures correctly. I tried using the same syntax from linuxserver, mouton99/nzbget-ng:arm64v8-latest (which I tested against their image and it worked properly), but it's not available on your image.
> 
> I am indeed only building and publishing for `linux/arm64`, not any other arch.
> 
> What you can do is simply git clone [my fork](https://github.com/Mouton99/docker-nzbget-ng), then run `docker build -t nzbget-ng .` from within that folder. That will give you a local image that you can use with `docker run ... nzbget-ng`

@Mouton99

I surely can do that, but it so happens that I'm also trying build for `linux/arm64v8`, but it seems that your image seems to be assign me `amd`, which, of course, doesn't work for me, I checked that with `Docker inspect`, for the package's architecture.

I can't say for sure that this is on the Jenkinsfile, but if it is, I'd be more than happy to submit a PR specifying docker to always fetch either `linux/nzbget-ng:arm64v8-latest` or `linux/nzbget-ng:arm64v8-testing`, whichever you're maontaining!

**Comment by x86txt** [comment]
If anyone wants a linux/amd64 build of @Mouton99's repo, you can find it here: https://hub.docker.com/repository/docker/x86txt/nzbget-ng/general

I'll update it as mouton909's pushes new updates to his repo.

**Comment by pannal** [comment]
> If anyone wants a linux/amd64 build of @Mouton99's repo, you can find it here: https://hub.docker.com/repository/docker/x86txt/nzbget-ng
> 
> I'll update it as mouton909's pushes new updates to his repo.

I think you meant arm64, right? As @Mouton99's image already is amd64

**Comment by x86txt** [comment]
Yep, I built it for both actually. The GitHub Action is the same either way.
